### PR TITLE
LPS-156566 | master

### PR DESCRIPTION
**Pull request by epark18**
https://issues.liferay.com/browse/LPS-156566

### CODE REVIEW

**Comment by epark18** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 1]
```diff
@@ -685,6 +685,64 @@ definition {
...
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 156");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionName = "Action Name",
+				active = "true",
+				scriptValidation = "println 'Success'",
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:718]: I think this groovy script is good, but how do you validate this `println` in the functional test?

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 2]
```diff
@@ -685,6 +685,64 @@ definition {
...
+
+			ObjectPortlet.deleteEntry(entry = "8");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 156");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.assertObjectActions(
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:739]: I think this assert not validate that the action is trigged

Comment (epark18) [reply modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:739]: Thank you for the comments, I will go review this test!

**Comment by epark18** [comment]
Will resend

**Pull request closed without merging**